doc/nrf/ug_nrf5340.rst

```diff
@@ -582,6 +582,44 @@ See the following instructions.
          Therefore, you must recover the network core first.
          Otherwise, if you recover the application core first and the network core last, the binary written to the application core is deleted and readback protection is enabled again after a reset.
 
+Debugging
+*********
+
+If your want to debug just the application core firmware, you need a single debug session.
+You can set it up as described in `Debugging nRF5340 with SES`_ or `Debugging an application`_ with Visual Studio Code.
+
+However if you want to debug the firmware running on the network core you need to set up two separate debug sessions:
+
+* A debug session with the application core.
+* A debug session with the network core.
+
+You can use the links above to set up each session.
+Make sure to select the appropriate CPU for debugging: the nRF5340 application core or the nRF5340 network core.
+When both sessions are established, first run the code execution on the application core.
+Its startup code will release the "NETWORK.FORCEOFF" signal to start the network core and it will allocate the necessary GPIO pins for the network core.
+Then you can start code execution on the network core in the other debug session.
+
+.. figure:: /images/debugging_ozone_nrf5340_app.png
+     :alt: SEGGER Ozone debugging session for the nRF5340 application core
+
+     SEGGER Ozone debugging session for the nRF5340 application core
+
+.. figure:: /images/debugging_ozone_nrf5340_net.png
+     :alt: SEGGER Ozone debugging session for the nRF5340 network core
+
+     SEGGER Ozone debugging session for the nRF5340 network core
+
+.. note::
+  When debugging if you want to reset the network core remember to reset and run code execution on the application core first.
+
+.. note::
+  You can also use alternative tools for debugging:
+ 
+  * SEGGER Ozone.
```

Comment:
Should it be `Visual Studio Code` since you described debugging with SEGGER Ozone above ?

Reply:
Good point. I think this is already resolved through the doc review from @divipillai. The thing is, we don't want to describe the very details of setting up the 2 debug sessions with the recommended toolchains as this is either hard (VS Code, and they plan to automate it) or we are not sure it's possible (SES). So we mention those tools as an alternative.